src/umpire/ResourceManager.cpp

```diff
@@ -177,7 +177,7 @@ ResourceManager::initialize()
   UMPIRE_LOG(Debug, "() leaving");
 }
 
-std::shared_ptr<strategy::AllocationStrategy>&
+strategy::AllocationStrategy*&
```

Comment:
@davidbeckingsale - Why does this (and some others) return a reference to a pointer instead of just a pointer?